pkg/deploy/generator/resources_rp.go

```diff
@@ -376,34 +376,35 @@ func (g *generator) rpVMSS() *arm.Resource {
 	}
 
 	for _, variable := range []string{
-		"armClientId",
-		"mdmFrontendUrl",
-		"mdsdEnvironment",
 		"acrResourceId",
 		"adminApiClientCertCommonName",
 		"armApiClientCertCommonName",
+		"armClientId",
+		"azureCloudName",
 		"billingE2EStorageAccountId",
 		"clusterMdmAccount",
 		"clusterMdsdAccount",
 		"clusterMdsdConfigVersion",
 		"clusterMdsdNamespace",
 		"clusterParentDomainName",
+		"databaseAccountName",
+		"dbtokenClientId",
```

Comment:
Would it make sense to split this into two commits, the first one just reordering these strings, and the second one adding the new value (and making the other changes)?

Reply:
Seems pretty small to split into it's own PR as it's only this reordering.  I can put it in another commit in this PR if that makes more sense.

Reply:
Yeah, two commits in the same PR is what I was suggesting.

Reply:
Misread it my bad :+1:   Will split.

Reply:
Done